.github/sync.yml

```diff
@@ -109,5 +109,7 @@ group:
       dest: code_of_conduct.md
     - source: assets/box_images/
       dest: assets/box_images/
+    - source: assets/style.css
```

Comment:
Actually wait, are you sure you want the main style file of this repo to replace your anvil style? I believe the default style is DaSL?

Reply:
@KatherineCox

Reply:
@cansavvy Yeah, since having 2 css files isn't an option right now, I decided to set up our AnVIL `style.css` so that it's identical to OTTR's style.css, but has any AnVIL-specific stuff at the bottom.  I'll just need to handle the merge manually and make sure that bottom section doesn't get deleted by the incoming PRs.